scripts/sct_propseg.py

```diff
@@ -437,22 +437,27 @@ def get_parser():
     sct.run(cmd, verbose)
 
     # extracting output filename
-    path_fname, file_fname, ext_fname = sct.extract_fname(input_filename)
-    output_filename = file_fname + "_seg" + ext_fname
+    file_seg = file_data + "_seg" + ext_data
```

Comment:
there are pros/cons, the cons is that the display at the end will be less aesthetic:
~~~
sct.printv("fslview "+input_filename+" "+output_name+" -l Red -b 0,1 -t 0.7 &\n", verbose, 'info')
~~~